app/models/service_template_container_template.rb

```diff
@@ -0,0 +1,94 @@
+class ServiceTemplateContainerTemplate < ServiceTemplateGeneric
+  def self.default_provisioning_entry_point(_service_type)
+    '/Service/Generic/StateMachines/GenericLifecycle/provision'
+  end
+
+  # create ServiceTemplate and supporting ServiceResources and ResourceActions
+  # options
+  #   :name
+  #   :description
+  #   :service_template_catalog_id
+  #   :config_info
+  #     :provision
+  #       :dialog_id (or)
+  #       :new_dialog_name
+  #       :container_project_id
+  #       :new_project_name
```

Comment:
if you are going to support `new_project_name`, you need to have `create_project` logic in the code which I cannot find.